tests/smoke/index.js

```diff
@@ -0,0 +1,11 @@
+const execFileSync = require('child_process').execFileSync;
+const path = require('path');
+
+const runTests = directoryName =>
+  execFileSync(path.join(__dirname, directoryName, 'index.js'), {
+    stdio: 'inherit',
+  });
+
+runTests('react-v14');
```

Comment:
I would not test v14 (I would only test stable + last major version), but I would add a test maybe on the @next tag of React, that can be interesting.